client/src/components/Message.js

```diff
@@ -52,7 +52,7 @@ const Message = ({ currentUser, error, errorMessage, children }) => {
   // Redux
   const { username } = useSelector(state => state.auth.user);
 
-  const regex = new RegExp("^@" + username + "$", "g");
+  const regex = new RegExp("^@" + username + "$", "i", "g");
```

Comment:
Separating the "i" and "g" tags gets rid the "g" tag. To fix this, combine them together in a single quotation with no commas as follows:
`const regex = new RegExp("^@" + username + "$", "gi");`